pkg/services/signingkeys/signingkeystore/store.go

```diff
@@ -39,7 +39,7 @@ type Store struct {
 type SigningKey struct {
 	ID         int64                   `json:"-" xorm:"id" db:"id"`
 	KeyID      string                  `json:"key_id" xorm:"key_id" db:"key_id"`
-	PrivateKey []byte                  `json:"private_key" xorm:"private_key" db:"private_key"`
+	PrivateKey string                  `json:"private_key" xorm:"private_key" db:"private_key"`
```

Comment:
I have changed this as well, but I haven't found any usage of `signingkeystore.SigningKey` type anywhere. If it's not used, can we remove this struct altogether?

Reply:
Yep seems unused, you can remove it :+1:

Reply:
Thanks, I'll do that in the next PR.

Reply:
https://github.com/grafana/grafana/pull/102041